Framework/PythonInterface/test/python/plugins/algorithms/VesuvioCorrectionsTest.py

```diff
@@ -27,6 +31,15 @@ def setUp(self):
         if self._test_container_ws is None:
             self.__class__._test_container_ws = create_test_container_ws()
 
+        self._input_bins = self._test_ws.blocksize()
+
+        # Check OS for RHEL7 or Ubuntu
+        dist = platform.linux_distribution()
```

Comment:
The only thing wrong with this is that `RHEL 6` passes with the values for `Windows`/`OSX` but perhaps it would be better to only have an exception for that? e.g.
```
dist = platform.linux_distribution()
if any(dist):